src/mds/mdstypes.h

```diff
@@ -1029,6 +1029,8 @@ struct fnode_t {
   frag_info_t fragstat, accounted_fragstat;
   nest_info_t rstat, accounted_rstat;
   utime_t rstat_dirty_from;
+  utime_t last_rstat_dirty_from;
+  bool rstat_dirty_from_delivered = false;
```

Comment:
just for debug purpose?

Reply:
@ukernel Hi, thanks for your review:-)

These two fields are used for reliable rstat_dirty_from delivery, but it seems that I lost some code here, will add those codes:-) Sorry.